src/core/CodeSnippet/CodeSnippet.stories.tsx

```diff
@@ -2,6 +2,104 @@ import React from "react";
 import { Meta, StoryObj } from "@storybook/react";
```

Comment:
TLDR for this file: pulled out common code snippets to reduce duplication, added stories for the new custom ordering and showCodeLines functionality